File: src/VSServiceStack/ProjectTemplates/ServiceStack.CSharp.MVC/ServiceModel/Hello.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using ServiceStack;

namespace ServiceStack.CSharp.MVC.ServiceModel
{
    public class Hello
    {
        public string Name { get; set; }
    }

    public class HelloResponse
    {
        public string Result { get; set; }
    }
}

File: src/ServiceStackVS/ProjectTemplates/Aurelia/Host/src/resources/elements/hello.ts
import {bindable, autoinject} from "aurelia-framework";
import {JsonServiceClient} from "servicestack-client";
import {Hello} from "../../dtos";

@autoinject()
export class HelloCustomElement {
    result: string;
    @bindable name: string;
    client: JsonServiceClient;

    constructor() {
        this.client = new JsonServiceClient('/');
    }

    nameChanged(newValue) {
        if (newValue.length > 0) {
            var req = new Hello();
            req.Name = newValue;
            this.client.get(req).then((helloResponse) => {
                this.result = helloResponse.Result
            });
        } else {
            this.result = '';
        }
    }
}